mlflow/langchain/_langchain_autolog.py

```diff
@@ -343,6 +344,13 @@ def __init__(self, *args, **kwargs):
         else:
             input_data = input_example
         try:
+            # we do not convert stream output iterator for logging as tracing
+            # will provide much more information than this function, we might drop
+            # this in the future
+            if func_name == "stream":
+                raise MlflowException(
+                    f"Unsupported function {func_name} for logging inputs and outputs."
+                )
```

Comment:
Instead of raising an error, can we log a message and skip logging inputs and outputs?

Reply:
It's within a try catch so it will go to the warning in except part

Reply:
`Failed to log inputs and outputs into` sound a bit weird because we don't even attempt.